Problem3/MethodProblem3.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Numerics;
+
+namespace SimpleProblem3
+{
+    public class MethodProblem3
+    {
+        public bool ThereAreDuplicates(int[] a)
+        {
+            int gaussSum = 0, actualSum = 0;
+            BigInteger multiply = 1, factorial = 1;
+            int size = a.Length;
+            for (int i = 0; i < size; i++)
+            {
+                actualSum += a[i];
+                multiply *= a[i];
+                factorial *= (i + 1);
+            }
+            gaussSum = size * (size + 1) / 2;
+            if (gaussSum != actualSum)
+
+                return true;
+            else
+                if (multiply != factorial)
+                    return true;
+            return false;
+        }
```

Comment:
Good point there, when writing that program my mind was focused more on numbers(links, differences between some arrays, what sort of conditions solve the whole case) while it being 4-5 AM on Saturday morning. Once again, thank you for the tip. I realise now I'm a big fan of this code feedback, it's a good eye-opener.